inc/saicounter.h

```diff
@@ -78,6 +81,45 @@ typedef enum _sai_counter_attr_t
      */
     SAI_COUNTER_ATTR_LABEL,
 
+    /**
+     * @brief Enable/disable packet count
+     *
+     * @type bool
+     * @flags CREATE_ONLY
+     * @default true
+     */
+    SAI_COUNTER_ATTR_ENABLE_PACKET_COUNT,
+
+    /**
+     * @brief Enable/disable byte count
+     *
+     * @type bool
+     * @flags CREATE_ONLY
+     * @default true
+     */
+    SAI_COUNTER_ATTR_ENABLE_BYTE_COUNT,
+
+    /**
+     * @brief Object Type of the stat-id
+     *
+     * @type sai_object_type_t
+     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+     * @condition SAI_COUNTER_ATTR_TYPE == SAI_COUNTER_TYPE_SELECTIVE
+     */
+    SAI_COUNTER_ATTR_OBJECT_TYPE,
+
+    /**
+     * @brief Stat id list
+     *
+     * List of statistics enum mapped to this counter
+     *
+     * @type sai_s32_list_t
```

Comment:
if values are enums on this list, you can use it like this: "@type sai_s32_list_t sai_stats_mode_t" etc

Reply:
> if values are enums on this list, you can use it like this: "@type sai_s32_list_t sai_stats_mode_t" etc

Here values are enums from one of sai_router_interface_stat_t/sai_vlan_stat_t/sai_queue_stat_t/sai_tunnel_stat_t/sai_port_stat_t. Can I define like below ? 

 "@type sai_s32_list_t sai_stat_id_t" or 
 "@type sai_u32_list_t sai_stat_id_t" since sai_stat_id_t typedef to uint32_t

Reply:
@kcudnik 
Above "@type" settings are failing in meta check. What is the correct way to define this ? In this case, enum type can be  one from different sai objects and we differentiate based on attr "SAI_COUNTER_ATTR_OBJECT_TYPE".

Reply:
What error do you get ?

Reply:
> What error do you get ?

With "@type sai_s32_list_t sai_stat_id_t",
```
ERROR: invalid enum list specified 'sai_s32_list_t sai_stat_id_t' on SAI_COUNTER_ATTR_STAT_ID_LIST
```
With "@type sai_u32_list_t sai_stat_id_t",
```
ERROR: invalid type '' for SAI_COUNTER_ATTR_STAT_ID_LIST
```

Reply:
> Because stat Id is not an enum its uint32, you can only specify enums, it that's not specific enums and could be different then you can leave list s32

Thanks @kcudnik . I will retain the original definition, "@type sai_s32_list_t"